### DISCUSSION TITLE
lesson 7:(6:34:40),Compile Error

### DISCUSSION THREAD

**Discussion by sako013** [smartcontractkit/full-blockchain-solidity-course-py#1479] [Q&A] [answered]
New compatible solc version available: 0.8.12
Compiling contracts...
  Solc version: 0.8.12
  Optimizer: Enabled  Runs: 200
  EVM Version: Istanbul
CompilerError: solc returned the following errors:

ParserError: Source "C:/Users/Sako/.brownie/packages/OpenZeppelin/openzeppelin-contracts@4.6.0/contracts/ownership/Ownable.sol" not found: File not found. Searched the following locations: "".
 --> contracts/Lottery.sol:5:1:
  |
5 | import "@openzeppelin/contracts/ownership/Ownable.sol";
 

i also have my .Vscode/setting.json

{
    "solidity.compileUsingRemoteVersion": "v0.8.8+commit.dddeac2f",
    "solidity.enableLocalNodeCompiler": false,
    "solidity.defaultCompiler": "remote",
    "solidity.remappings": [
        "@chainlink/=/Users/Sako/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@0.2.2",
        "@openzeppelin/=/Users/Sako/.brownie/packages/OpenZeppelin/openzeppelin-contracts@4.6.0"
    ]
}

https://github.com/smartcontractkit/full-blockchain-solidity-course-py/blob/main/chronological-issues-from-video.md#lesson-4

**Comment by cromewar** [comment, marked as answer]
Hello @sako013 
The correct path is not `ownership` is `access` as you can see bellow.
![image](https://user-images.githubusercontent.com/20199180/168475514-d95f5e69-f1d0-423d-83ab-7754d6497e42.png)
